rockcraft.yaml

```diff
@@ -0,0 +1,29 @@
+name: kratos
+base: bare
+build-base: ubuntu:22.04
+version: "0.13.0"
+summary: Ory Kratos identity server
+description: |
+  Ory Kratos is a Identity, User Management and Authentication system for the Cloud.
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  kratos:
+    override: replace
+    command: kratos serve
+    startup: enabled
```

Comment:
Will it not interfere with pebble's [services](https://github.com/canonical/kratos-operator/blob/main/src/charm.py#L195) or replace the service params specified in kratos?

Reply:
We use replace in the kratos-operator, so this will be overwritten. This is the same entrypoint that the kratos Dockerfile has. Tbh I have no problem with removing this, as it will always fail (there is no config file).

Reply:
I'm fine with both, I leave it up to you then

Reply:
I will leave it like this to follow upstream.